agent.go

```diff
@@ -18,6 +18,7 @@ import (
 	"syscall"
 	"time"
 
+    _ "github.com/opencontainers/runc/libcontainer/nsenter"
```

Comment:
This is not indented the right way, and it's not in the right place in the code I think.
Could run some linters to fix that please:
```bash
go_packages=$(go list ./... | grep -v protocols | grep -v vendor)
go_files=`go list -f  '{{.Dir}}/*.go' $go_packages`
go fmt $go_packages
golint $go_packages
go vet $go_packages
gofmt -s -w $go_files
gocyclo -over 15 $go_files
ineffassign .
```

Reply:
Sure. Quickly.